erizo_controller/erizoClient/src/Connection.js

```diff
@@ -71,6 +75,15 @@ Erizo.GetUserMedia = function (config, callback, error) {
     if (config.screen) {
         L.Logger.debug('Screen access requested');
         switch (Erizo.getBrowser()) {
+            case 'electron' :
+                 L.Logger.debug('Screen sharing in Electron');
+                var screenConfig = {};
+                screenConfig.video = config.video || {};
+                screenConfig.video.mandatory = config.video.mandatory || {};
+                screenConfig.video.mandatory.chromeMediaSource = 'screen';
+                var theConfig = {video: { mandatory: { chromeMediaSource: 'screen' } } };
```

Comment:
are we using this var? seems like we're building the same object twice but we just use one.